packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -82,8 +82,21 @@ const Verification: FC<TVerificationProps> = ({ selectedJurisdiction }) => {
     const { data: settings, update: updateSettings } = useSettings();
     const { submitIDVDocuments } = useIdentityDocumentVerificationAdd();
     const { getModalState, hide, show } = useModal();
+    const [isDetailsVerified, setIsDetailsVerified] = useState(false);
     const { isMobile } = useDevice();
 
+    const screens: ScreensType = {
```

Comment:
If possible this should not be inside the parent component `Verification` and should be outside, the components will be recreated every single render

Reply:
Updated implementation, please re-review 👍